tools/mo/openvino/tools/mo/utils/ir_reader/extenders/parameter_extender.py

```diff
@@ -26,7 +26,7 @@ def extend(op: Node):
                     shape[i] = -1
                     if ".." in dim:
                         has_shapes_with_boundaries = True
-            shape = shape_array([d if d != -1 else dynamic_dimension_value for d in shape])
+            shape = shape_array([d if d not in [-1, '?'] else dynamic_dimension_value for d in shape])
```

Comment:
please note that deserialization does not save `shape` attribute value in original form. because `?` is replaced with `-1`. On the other hand, `-1` and `?` are equivalent and not so important to save it in the original form